File: OnlineBooking.API/OnlineBooking.API/Models/BookingModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OnlineBooking.API.Models
{
    public class BookingModel
    {
        public string name { get; set; }        
        public string email { get; set; }        
        public string phone { get; set; }
        public DateTime date { get; set; }
        public DateTime startTime { get; set; }
        public DateTime endTime { get; set; }
    }
}

File: OnlineBooking.Client/src/app/model/bookingmodel.ts
export interface BookingModel {
        Id: string; 
        Name:string;
        StartTime: Date;
        EndTime: Date;
        Description: string;
        AllDay:boolean;
        Recurrence: boolean;
        Categorize: string;        
        Email:string;
        Phone: string;  
        Status:number;    
    
}